### PR TITLE
Remove duplicate `flex-basis` declarations

### PR DESCRIPTION
**Pull request by simonsmith**

### CODE REVIEW

File: lib/size-lg.css [hunk 1]
```diff
@@ -38,68 +38,71 @@
...
    * Supports: 2, 3, 4, 5, 6, 8, 10, 12 part
    *
-   * Uses `flex-basis: auto` with a width to avoid box-sizing bug in IE10/11 - http://git.io/vllMD
+   * Use `flex-basis: auto` with a width to avoid box-sizing bug in IE10/11
+   * http://git.io/vllMD
    */
 
-  .u-lg-size1of12 {
+  /* postcss-bem-linter: ignore */
+  [class*="u-lg-size1"],
```
Comment (giuseppeg) [inline lib/size-lg.css]: Can't we use  `[class*="u-lg-size"]` to match them all ?

Comment (simonsmith) [inline lib/size-lg.css]: You are 100% correct. That'll teach me for opening PRs before bed.

Comment (simonsmith) [inline lib/size-lg.css]: Updated

File: lib/size.css [hunk 2]
```diff
@@ -38,68 +37,63 @@
  * Intentional redundancy build into each set of unit classes.
  * Supports: 2, 3, 4, 5, 6, 8, 10, 12 part
  *
- * Uses `flex-basis: auto` with a width to avoid box-sizing bug in IE10/11 - http://git.io/vllMD
+ * Use `flex-basis: auto` with a width to avoid box-sizing bug in IE10/11
+ * http://git.io/vllMD
  */
 
-.u-size1of12 {
+/* postcss-bem-linter: ignore */
```
Comment (giuseppeg) [inline lib/size.css]: Will this ignore just this rule?

Comment (simonsmith) [inline lib/size.css]: Yep, just the ruleset directly after

File: lib/size-lg.css (hunk 1 above)
Comment (simonsmith) [inline lib/size-lg.css]: Seems this effects the sizeFit and Full rules as well. Going to revert to being more explicit as before

**Pull request merged**